bindings/shmem_bind_c.m4

```diff
@@ -11,11 +11,27 @@ define(`SHMEM_BIND_C_RMA', dnl args: macro_name, end
 `$1(float,      float)$2
 $1(double,     double)$2
 $1(longdouble, long double)$2
+$1(schar,      signed char)$2
```

Comment:
Extremely minor, but perhaps `char` could be before `signed char` to match the spec table and the unit tests...